docs/content/hdfs.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "HDFS Remote"
+description: "Remote for Hadoop Distributed Filesystem"
+---
+
+{{< icon "fa fa-globe" >}} HDFS
+-------------------------------------------------
+
+[HDFS](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html) is a distributed file-system, part of [Apache Hadoop](https://hadoop.apache.org/) framework.
+
+Paths are specified as `remote:` or `remote:path/to/dir`.
+
+Here is an example of how to make a remote called `remote`. First run:
+
+     rclone config
+
+This will guide you through an interactive setup process:
+
+```
+No remotes found - make a new one
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+name> hadoop
+Type of storage to configure.
+Enter a string value. Press Enter for the default ("").
+Choose a number from below, or type in your own value
+[snip]
+XX / Hadoop distributed file system
+   \ "hdfs"
+[snip]
+Storage> hdfs
+hadoop name node and port
+Enter a string value. Press Enter for the default ("").
+Choose a number from below, or type in your own value
+ 1 / Connect to host namenode at port 8200
+   \ "namenode:8020"
+namenode> namenode.hadoop:8020
+Edit advanced config? (y/n)
+y) Yes
+n) No (default)
+y/n> n
+Remote config
+--------------------
+[hadoop]
+type = hdfs
+namenode = namenode.hadoop:8020
+--------------------
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+Current remotes:
+
+Name                 Type
+====                 ====
+hadoop               hdfs
+
+e) Edit existing remote
+n) New remote
+d) Delete remote
+r) Rename remote
+c) Copy remote
+s) Set configuration password
+q) Quit config
+e/n/d/r/c/s/q> q
+```
+
+This remote is called `remote` and can now be used like this
+
+See all the top level directories
+
+    rclone lsd remote:
+
+List the contents of a directory
+
+    rclone ls remote:directory
+
+Sync the remote `directory` to `/home/local/directory`, deleting any excess files.
+
+    rclone sync -i remote:directory /home/local/directory
+
+### Modified time ###
+
+Store time accurate to 1 second.
+
+### Checksum ###
+
+No checksums are implemented.
+
+### Usage information ###
+
+You can use the `rclone about remote:` command which will display filesystem size and current usage.
+
+### Restricted filename characters
+
+In addition to the [default restricted characters set](/overview/#restricted-characters)
+the following characters are also replaced:
+
+| Character | Value | Replacement |
+| --------- |:-----:|:-----------:|
+| :         | 0x3A  | ：           |
+
+Invalid UTF-8 bytes will also be [replaced](/overview/#invalid-utf8).
+
+### Limitations ###
+
+* No server-side `Move`.
+* Checksums not implemented.
+* Hadoop user name can not be configured, but specified with environment variable `HADOOP_USER_NAME`
```

Comment:
Could we set the environment variable from the rclone config?

Reply:
done in new commit

Reply:
Great